tests/models/index.js

```diff
@@ -21,12 +21,12 @@ const allTests = async () => {
     }
   }
   // await activityTests(app)
-  await documentTests(app)
-  await publicationTests(app)
+  // await documentTests(app)
+  // await publicationTests(app)
   await attributionTests(app)
-  await readerTests(app)
-  await noteTests(app)
-  await tagTests(app)
+  // await readerTests(app)
+  // await noteTests(app)
+  // await tagTests(app)
```

Comment:
Once you are all done, you can enable the document, publication, reader, note and tag tests again. Your changes shouldn't affect them.